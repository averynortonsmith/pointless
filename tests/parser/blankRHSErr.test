
foo = (1, _)