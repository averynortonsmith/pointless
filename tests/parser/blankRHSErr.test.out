-------------------------------------------------------------------------------
Parser Error:

Expected 'Number or String or Name or Label or Bool or LArray or LParen or LBracket', got 'Blank'
-------------------------------------------------------------------------------

At (line 2 column 11) in 'tests/parser/blankRHSErr.test'
foo = (1, _)
          ^
